client/public/css/index.css

```diff
@@ -181,6 +181,7 @@ body {
   left: 0;
   right: 0;
   bottom: 0;
+  font-size: 13px;
```

Comment:
Setting the font size on the editor container works fine; setting it via the Ace bindings is not currently working and I'll need to file a bug with `purescript-contrib/purescript-ace`.